modernizer-maven-plugin/src/main/java/org/gaul/modernizer_maven_plugin/ModernizerMojo.java

```diff
@@ -265,28 +276,28 @@ public void execute() throws MojoExecutionException {
         }
     }
 
-    private Collection<String> readExclusionsFile(String exclusionsFilePath)
```

Comment:
Is there a reason to change this method? I think that "exclusions file" still describes the annoation class file.